chord/chord.go

```diff
@@ -92,8 +93,8 @@ func NewChordNode(listener net.Listener, config ChordConfig) (*ChordNode, error)
 	go chordNode.chordServer.Serve(listener)
 
 	// TODO replace by a constant or config.fixFingerInterval
-	go chordNode.RunAtInterval(StabilizeDaemon, 5)
-	go chordNode.RunAtInterval(FixFingersDaemon(chordNode), 5)
+	go chordNode.RunAtInterval(StabilizeDaemon, 4)
+	go chordNode.RunAtInterval(FixFingersDaemon(chordNode), 6)
```

Comment:
Did you observed any issue with 5s and 5s interval?

Reply:
We'll be replacing those constants with configurable variables, so it shouldn't matter. But yes, I changed those because I thought that could help me improve something. It seemingly didn't.